PhysicsTools/PatAlgos/python/slimming/miniAOD_tools.py

```diff
@@ -1,8 +1,11 @@
 import FWCore.ParameterSet.Config as cms
 
 from PhysicsTools.SelectorUtils.tools.vid_id_tools import *
+from Configuration.StandardSequences.Eras import eras
+
 
 def miniAOD_customizeCommon(process):
+    process.load("PhysicsTools.PatAlgos.slimming.era_customizers")
```

Comment:
I will improve it when I'll understand the more proper way to properly integrate eras and customises ( @Dr15Jones )